stock_cycle_count/models/stock_move_line.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    line_accuracy = fields.Float(
```

Comment:
I'm wondering if accuracy represent the accuracy on all related move lines or only for inventory related move lines ?

@ArnauCForgeFlow

Reply:
Hey @rousseldenis  this line_accuracy is only for inventory related move lines